geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/filter/FilterTest.scala

```diff
@@ -95,7 +96,7 @@ class FilterTester extends Specification with TestWithDataStore with LazyLogging
     val filterCount = mediumDataFeatures.count(filter.evaluate)
     val query = new Query(sftName, filter)
     Option(projection).foreach(query.setPropertyNames)
-    val queryCount = fs.getFeatures(query).size
+    val queryCount = SelfClosingIterator(fs.getFeatures(query)).length
```

Comment:
by default we return an estimated count for .size - so if you want the real count, you have to either set a query hint or a sys property or just count them (which is what i'm doing here)